charts/tidb-lightning/values.yaml

```diff
@@ -0,0 +1,41 @@
+# Default values for tidb-lightning.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image: pingcap/tidb-lightning:v3.0.1
+imagePullPolicy: IfNotPresent
+service:
+  type: NodePort
```

Comment:
Does lightning need external accessing?

Reply:
There is a webpage to show current job progress.

Reply:
@kennytm Seems this webpage is broken.

Reply:
(Tracking issue: pingcap/tidb-lightning#213)

Reply:
@kennytm What does this mean? Did you mean it gets fixed in that PR? I've tried v3.0.2 which contains this PR, but still got the same broken page.

Reply:
Sorry just a typo.